src/kestrel_datasource_stixshifter/interface.py

```diff
@@ -195,7 +196,7 @@ def query(uri, pattern, session_id, config, store=None):
             check_module_availability(connector_name)
 
             data_path_striped = "".join(filter(str.isalnum, profile))
-            ingestfile = ingestdir / f"{i}_{data_path_striped}.json"
+            ingestfile = ingestdir / f"{i}_batch_index_{data_path_striped}.json"
```

Comment:
Shall we just create a function `make_ingest_stixbundle_filepath(batch_index)` instead of manipulating the string through the module? If someone forget to replace substring in the future when using `ingestfile`, that may cause issue.